frontend/src/views/Market.vue

```diff
@@ -2288,6 +2325,10 @@ export default Vue.extend({
       return input[0].toUpperCase() + input.slice(1);
     },
 
+    formatUrl(input: string): string {
+      return `<a href="${getConfigValue('blockExplorerUrls')[0]}/tx/${input}">${input.substr(0,4)}...${input.substr(-6)})}</a>`;
```

Comment:
Why are you returning template here instead of using template at the top and setting :href value dynamically?

Reply:
sorry, im not that good with FE

Reply:
You can place <a> tag in the upper part (template) and set :href="" to the value you want

Reply:
to be honest, idk how xD